### PR TITLE
Relicense signing Shourrya Guha

### PR DESCRIPTION
**Pull request by ShourryaGuha**
Discord: Shourrya#2094
Email: guhashourrya@gmail.com

### CODE REVIEW

File: RELICENSE/ShourryaGuha.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+```md
```
Comment (hzhou0) [inline RELICENSE/ShourryaGuha.md]:
```suggestion

File: RELICENSE/ShourryaGuha.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+chosen by the chrysalis-blc project team.
+
+A portion of the commits made by the Github handle "ShourryaGuha", with commit author 
+"ShourryaGuha", are copyright of Shourrya Guha. This document hereby grants the
+chrysalis-blc project team to relicense chrysalis-blc code, including all past, present and
+future contributions of the author listed above.
+
+Shourrya Guha 
+creation date of document (format: 2022/01/30)
+```
```
Comment (hzhou0) [inline RELICENSE/ShourryaGuha.md]:
```suggestion
```
```

**Review by hzhou0 — CHANGES_REQUESTED** [review]
Just remove these two lines

**Comment by ShourryaGuha** [comment]
Done with the changes

**Pull request merged**